README.md

```diff
@@ -13,21 +13,9 @@ Codesign Electron macOS apps
 ## Installation
 
 ```sh
-# For use in npm scripts
-npm install --save @electron/osx-sign
-# yarn
-yarn add @electron/osx-sign
+npm install --save-dev @electron/osx-sign
 ```
 
-```sh
-# For use from CLI
-npm install -g @electron/osx-sign
-# Yarn
-yarn global add @electron/osx-sign
-```
-
-*Note: `@electron/osx-sign` is a dependency of [`@electron/packager`](https://github.com/electron/packager) as of 6.0.0 for signing apps on macOS. However, feel free to install this package globally for more customization beyond specifying identity and entitlements.*
```

Comment:
Maybe adjust the note so it says something like " If you're using Electron Packager or Electron Forge, this functionality is built-in."

Reply:
[0416fc6](https://github.com/electron/osx-sign/pull/317/commits/0416fc698a9cd6643f98f20ec406580db09c5380)